### ISSUE TITLE
Result on 1k

### ISSUE THREAD

**Issue by seominjoon** [therne/dmn-tensorflow#1]
Would you be able to report the result on the 1k dataset as well? DMN+ paper doesn't report the 1k result so I was always wondering.. Thanks for great work!

**Comment by therne** [comment]
I haven't tested yet because it need extra hyperparameter optimization to prevent overfitting on smaller data - unlike the original DMN 😢 but I'll try and upload it soon!

**Comment by tyler7585** [comment]
Would you please upload your training model? I just download your code and follow the instructions , but no matter how I tried to train, the accuracy is much lower than what you said. I guess it might be some problem.

**Comment by therne** [comment]
@tyler7585 Unfortunately I've deleted all of my trained models 😢 and It'll take much time to train again and upload the entire models.

Can you tell me the task numbers that you're in trouble? It'll be much faster to upload those tasks only. (personally #19 was hard to reach that result - trained about 15 times and achieved it)

**Comment by tyler7585** [comment]
Would you please just tell me how much times it will take to train for the first task, the accuracy  just fluctuate between 19~21. I have trained it for 2 times(256 epoch per time). So I thought maybe there is a problem, and do not train it any more.

**Comment by tyler7585** [comment]
You said, #19 task trained about 15 times, so every time you train, it will update the train model based on the last train model? Am I right?

**Comment by tyler7585** [comment]
![2016-07-20-084848_797x573_scrot](https://cloud.githubusercontent.com/assets/20261535/16970422/d6ba8f98-4e56-11e6-94bf-a48a22db4f5e.png)

**Comment by tyler7585** [comment]
BTW, I am training with your default setting.

**Comment by tyler7585** [comment]
@therne Is this situation normal? Because I don't find it has a clear increase.

**Comment by tomsonsgs** [comment]
@tyler7585 do you solve the problem?i meet the same problem
